src/Doctrine/EchoSQLLogger.php

```diff
@@ -0,0 +1,28 @@
+<?php declare(strict_types=1);
+
+namespace Frosh\DevelopmentHelper\Subscriber;
```

Comment:
The namespace should be `Frosh\DevelopmentHelper\Doctrine`.

Reply:
Can you make a Pull Request?

Reply:
Done in #31